husarion_ugv_lights/config/lights_controller_parameters.yaml

```
@@ -0,0 +1,17 @@
+lights_controller:
+  animations_config_path:
+    type: string
+    default_value: ""
+    description: Path to a YAML file with a description of led configuration. This file includes definition of robot panels, virtual segments and default animations.
```

In the read me was `LEDs`
```suggestion
    description: Path to a YAML file with a description of LED configuration. This file includes definition of robot panels, virtual segments and default animations.
```